libs/blocks/tabs/tabs.js

```diff
@@ -57,7 +57,28 @@ export function getRedirectionUrl(linkedTabsList, targetId) {
   return currentUrl;
 }
 
-function changeTabs(e) {
+const generateStorageName = (tabId) => {
+  const { pathname } = window.location;
+  return `${pathname}/${tabId}-tab-state`;
+};
+
+const loadActiveTab = (config) => {
+  if (config['remember-selected-tab'] !== 'on') return 0;
```

Comment:
We need to check if they prefer an explicit long token name such as `remember-selected-tab` which might be challenging to remember, or a shorthand version which can be adapted later in other places, aka : `remember`